lab-nassir/package.json

```diff
@@ -0,0 +1,51 @@
+{
+  "name": "lab-21-22-cowsay-client",
+  "version": "1.0.0",
+  "description": "![cf](https://i.imgur.com/7v5ASc8.png) lab 21-22 cowsay client ======",
+  "main": "webpack.config.js",
+  "scripts": {
+    "build": "./node_modules/webpack/bin/webpack.js",
+    "build-watch": "./node_modules/webpack-dev-server/bin/webpack-dev-server.js --inline --hot",
+    "lint": "./node_modules/eslint/bin/eslint.js  ./",
```

Comment:
Great job writing the script, but you don't have eslint as a dependency and so this doesn't work :P